.travis.yml

```diff
@@ -1,11 +1,28 @@
+sudo: required
+dist: trusty
+
 language: node_js
 node_js:
   - "4.1"
+python:
+  - "3.4"
 
 cache:
   pip: true
   directories:
     - node_modules
 
+before_install:
+  - sudo apt-get -y install python3-pip python-dev
+  - sudo pip3 install -U pip setuptools virtualenvwrapper
+
+  - npm install -g bower
+  - bower install
```

Comment:
I need sudo to get pip3 . Container builds don't have it as pip3 isn't on the white list

-----Original Message-----
From: "Lasse Schuirmann" <notifications@github.com>
Sent: ‎23-‎04-‎2016 16:52
To: "coala-analyzer/coala-html" <coala-html@noreply.github.com>
Cc: "AbdealiJK" <abdealikothari@gmail.com>; "Mention" <mention@noreply.github.com>
Subject: Re: [coala-analyzer/coala-html] Add testing (#41)

In .travis.yml:
>  
>  cache:
>    pip: true
>    directories:
>      - node_modules
>  
> +before_install:
> +  - sudo apt-get -y install python3-pip python-dev
> +  - sudo pip3 install -U pip setuptools virtualenvwrapper
> +
> +  - npm install -g bower
> +  - bower install
can't you get this with the apt: thing provided by travis?
—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub